scripts/change_address_types.py

```diff
@@ -0,0 +1,47 @@
+# encoding=utf-8
+
+from sefaria.helper.schema import *
+
+def change_mishneh_torah():
+    '''
+    changes addressTypes of all Mishneh Torah indexs to ['Perek' , 'Halkhah']
+    :return:
+    '''
+
+    yad_list = library.get_indexes_in_category(u"Mishneh Torah")
+    schema_yad_dict = {}
+    for title in yad_list:
+        schema_yad_dict[title] = library.get_schema_node(title)
+
+    for node in schema_yad_dict.values():
+        if len(node.sectionNames) != 2:
```

Comment:
what are you checking here? should depth one nodes get an address type of just 'Perek'? I don't know of examples